### DISCUSSION TITLE
How can I use Row Level Security in Superset version 1.1.0?

### DISCUSSION THREAD

**Discussion by norpriest** [apache/superset#16657] [Q&A / Help] [answered]
Hi,

In version 0.0.0dev, there was this option to set up RLS, but in version 1.1.0 there's none.

How can I use RLS in Superset version 1.1.0?

![lpo](https://user-images.githubusercontent.com/72668447/132790055-991e1c82-75dc-4b45-9862-215afcbd80f3.png)

**Comment by villebro** [comment, marked as answer]
RLS is enabled by default since 1.2.0. If you don't want to upgrade, check the `UPDATING.md` which states the following for 1.0.0:

> #11575 The Row Level Security (RLS) config flag has been moved to a feature flag. To migrate, add `ROW_LEVEL_SECURITY: True` to the `FEATURE_FLAGS` dict in `superset_config.py`.
